backend/mutations.py

```diff
@@ -525,6 +530,29 @@ def mutate(self, info, report_data=None):
         return CreateReportMutation(report=report)
 
 
+class AuthenticateUserMutation(graphene.Mutation):
+    user = graphene.Field(UserType)
+    success = graphene.Boolean()
+
+    class Arguments:
+        email = graphene.String()
+        password = graphene.String()
+
+    def mutate(self, info, email, password):
+        
+        def cryptPassword(password):
+            return password
```

Comment:
Nice! This will be perfect for later down the road

Reply:
Thank you!